docs/1.1-DRAFT/index.md

```diff
@@ -2013,6 +2013,53 @@ When generating the _RO-Crate Website_ from _RO-Crate JSON-LD_, the code MUST us
 
 Where there is no RDF ontology available, then implementors SHOULD attempt to provide context by creating stable web-accessible URIs to document properties and classes, for example, by linking to page describing an XML element or attribute from an XML schema, pending the publication of a formal ontology.
 
+
+### Adding new or ad hoc vocabulary terms
+
+Context terms must ultimately map to HTTP(s) URIs which poses challenges for crate-authors wishing to use their own vocabularies.
+
+RO-Crate provides some strategies To add a new term (a Class or Property) that is not in schema.org or another published vocabulary, so that there is a stable URI that can be added to the @context. 
+
+#### Choosing URLs for ad hoc terms
+
+For projects that have their own web-presence, URLs MAY defined and SHOULD resolve to useful content. For example for a project with web page <https://criminalcharacters.com/> the property `education` could have a URL: https://criminalcharacters.com/vocab/#education which resolves to an HTML page that explains the term.
+
+For ad hoc terms where the crate author does not have the resources to put up an HTML page, an ad-hoc URL MAY be used in the form `https://w3id.org/ro/terms/criminalcharacters/education` where `criminalcharacters` is acting as a _namespace_ for one or more related terms like `education`. Ad-hoc namespaces under `https://w3id.org/ro/terms/` are available on first-come-first-serve basis; to avoid clashes, namespaces SHOULD be registered by [submitting terms and definitions](https://github.com/ResearchObject/ro-terms) to the RO-Crate project. 
```

Comment:
Could the url be made to: https://w3id.org/ro/terms/criminalcharacters#education?

Reply:
The reason is that doing content negotiation based on full files may be easier to handle than doing CN to individual terms.

Reply:
I want to leave the example as it is a real one - there won't be any content negotiation but it is a reasonable strategy for someone to document their terms (and a machine readable version could be included in-page). 

Could you supplu an example that points to  https://w3id.org/ro/terms/ ?